src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldsets">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+          >
+        </label>
+
+        <div class="form-field">
+          <label>
+           How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+              required
+            >
+          </label>
+        </div>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            placeholder="dd.mm.yyyy"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkbox"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldsets">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldsets">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label >
+            Do you love cats?
+            <input
+              type="radio"
+              name="isLoveCats"
+              required
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="question"
```

Comment:
Be more attentive to ditails
```suggestion
              name="isLoveCats"
```